.github/ISSUE_TEMPLATE/0-bug-report.yml

```diff
@@ -3,6 +3,19 @@ description: Report an issue with the Salesforce Code Analyzer Extension
 title: "[BUG] <YOUR_TITLE_HERE>"
 labels: []
 body:
+- type: dropdown
+  attributes:
+    label: Have you tried to verified this is Salesforce Code Analyzer Extension specific?
+    description: |
+      Oftentimes, the the issue is not specific to this extension, but instead specific to our Salesforce Code Analyzer CLI. Before creating this issue, please check to see if the same issue appears when running the [Salesforce Code Analyzer CLI](https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer-command-reference.md). If so, then please report the issue in the [sfdx-scanner repo](https://github.com/forcedotcom/sfdx-scanner/issues) instead. Otherwise, you may create your issue here.
```

Comment:
This wording feels a little odd.
Perhaps:
```
Oftentimes, issues are not specific to this extension, and can be reproduced with the Salesforce CLI directly. Before creating this issue, please check to see if the same issue occurs when running the [Salesforce Code Analyzer CLI](https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/code-analyzer-command-reference.md) directly. if so, please report....
```

Reply:
Updated. Thanks!